### PR TITLE
newpackage/tippecanoe

### PR DESCRIPTION
**Pull request by Sinan81**
newpackage/tippecanoe: "Build vector tilesets from large collections of GeoJSON features."

### CODE REVIEW

File: var/spack/repos/builtin/packages/tippecanoe/package.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    url      = "https://github.com/mapbox/tippecanoe/archive/1.34.3.tar.gz"
+
+    version('1.34.3', sha256='7a2dd2376a93d66a82c8253a46dbfcab3eaaaaca7bf503388167b9ee251bee54')
+
+    depends_on('sqlite')
+    depends_on('zlib')
+
+    def edit(self, spec, prefix):
+        makefile = FileFilter('Makefile')
+        makefile.filter('PREFIX \?= /usr/local', 'PREFIX = ' + self.prefix)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/tippecanoe/package.py]:
```
var/spack/repos/builtin/packages/tippecanoe/package.py:23: [W605] invalid escape sequence '\?'
```
Just add an `r` before the string to make it a raw string: `r'...'`

Comment (Sinan81) [reply var/spack/repos/builtin/packages/tippecanoe/package.py]: should I both add 'r' and remove slash in '... \?= ...' or just add 'r'?

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/tippecanoe/package.py]: You need the backslash, just add the 'r'

Comment (Sinan81) [reply var/spack/repos/builtin/packages/tippecanoe/package.py]: Got it! Thanks.

**Pull request merged**